x/superfluid/types/errors.go

```diff
@@ -115,3 +117,19 @@ type UnexpectedDenomError struct {
 func (e UnexpectedDenomError) Error() string {
 	return fmt.Sprintf("provided denom (%s) was expected to be formatted as follows: %s", e.ProvidedDenom, e.ExpectedDenom)
 }
+
+type TokenConvertedLessThenDesiredStakeError struct {
+	ActualTotalAmtToStake   sdk.Int
+	ExpectedTotalAmtToStake sdk.Int
+}
+
+func (e TokenConvertedLessThenDesiredStakeError) Error() string {
+	return fmt.Sprintf("actual amount converted to stake (%s) is less then minimum amount expected to be staked (%s)", e.ActualTotalAmtToStake, e.ExpectedTotalAmtToStake)
+}
+
+type NoValsetNoSuperfluidDelegationError struct {
+}
+
+func (e NoValsetNoSuperfluidDelegationError) Error() string {
+	return fmt.Sprintf("No val-set for delegation nor existing superfluid delegation")
+}
```

Comment:
nit: what is the reason for defining the error this way when it has no custom fields (i.e. empty struct)?

Reply:
addressed in https://github.com/osmosis-labs/osmosis/pull/5949/commits/96f486b5e1bbf468406fc4b5d1aa3e17ef6b4003, actually removed the error since it turned out to be unneeded